docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -160,6 +160,65 @@ the dependencies as shown below.
     :start-after: [START main_flow]
     :end-before: [END main_flow]
 
+
+Re-using a decorated task
+-------------------------
+
+Decorated tasks are flexible. You can re-use a decorated task in multiple DAGs, overriding the operator
+parameters such as the ``task_id``, ``queue``, ``pool``, etc, as well as the task's arguments.
+
+Below is an example of how you can re-use a decorated task in multiple DAGs:
```

Comment:
```suggestion
Reusing a decorated task
-------------------------

Decorated tasks are flexible. You can reuse a decorated task in multiple DAGs, overriding the operator
parameters such as the ``task_id``, ``queue``, ``pool``, etc., as well as the task's arguments.

Below is an example of how you can reuse a decorated task in multiple DAGs:
```

Reply:
>as well as the task's arguments.
We're just overriding the operator parameters, right?

Reply:
Should we link to the [list of operator params](https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html#default-arguments) we have in the docs so users have some more reference to what can be overwritten albeit not exhaustive? WDYT?

Reply:
No. I feel that linking to the operator params may cause confusion/derailing...